ra/ra_test.go

```diff
@@ -1364,9 +1390,10 @@ func TestRegistrationKeyUpdate(t *testing.T) {
 // checkCertificatesPerNameRateLimit's FQDN exemption logic.
 type mockSAWithFQDNSet struct {
 	mocks.StorageAuthority
-	fqdnSet    map[string]bool
-	nameCounts *sapb.CountByNames
-	t          *testing.T
+	fqdnSet            map[string]bool
+	nameCounts         *sapb.CountByNames
+	issuanceTimestamps map[string]*sapb.Timestamps
```

Comment:
Feels vaguely weird that the mock data for the new mock method is completely divorced from the existing `fqdnSet` mock data -- a test could call `addFqdnSet` and it would affect the results of `FQDNSetExists` but not `FQDNSetTimestampsForWindow` -- but that's already the case for `nameCounts` and `CountFQDNSets` so I guess it's not the end of the world.

But couldn't `nameCounts` derive its data from this new `issuamceTimestamps`, simply by taking the len of each value? No need to duplicate information in the mock.

Reply:
> But couldn't nameCounts derive its data from this new issuamceTimestamps, simply by taking the len of each value? No need to duplicate information in the mock.

Good call, I went ahead and made this change.